regimes/se/tax_categories.go

```diff
@@ -0,0 +1,94 @@
+package se
+
+import (
+	"github.com/invopop/gobl/cal"
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/num"
+	"github.com/invopop/gobl/tax"
+)
+
+// Tax rates specific to Sweden.
+const (
+	TaxRateStandard       cbc.Key = "Normalskattesats"
+	TaxRateReduced        cbc.Key = "Skattesats 12 procent"
+	TaxRateHeavilyReduced cbc.Key = "Skattesats 6 procent"
+	TaxRateExempt         cbc.Key = "Momsfri"
+)
+
+var taxCategories = []*tax.CategoryDef{
+	{
+		Code: tax.CategoryVAT,
+		Name: i18n.String{
+			i18n.EN: "VAT",
+			i18n.SE: "Moms",
+		},
+		Title: i18n.String{
+			i18n.EN: "Value Added Tax",
+			i18n.SE: "Moms",
+		},
+		Retained: false,
+		Rates: []*tax.RateDef{
+			{
+				Key: tax.RateStandard,
+				Name: i18n.String{
+					i18n.EN: "Standard Rate",
+					i18n.SE: "Normalskattesats",
+				},
+				Values: []*tax.RateValueDef{
+					{
+						Percent: num.MakePercentage(250, 3),
+						Since:   cal.NewDate(1990, 1, 1),
+					},
+				},
+			},
+			{
+				Key: tax.RateReduced,
+				Name: i18n.String{
+					i18n.EN: "First Reduced Rate",
+					i18n.SE: "Skattesats 12 procent",
```

Comment:
Probably don't want numbers in the names as they may change in the future.

Reply:
Fair point. I'll stick to the translation from the English term then (current ones come directly from the Swedish Tax Agency's website)